boards/arm/beagle_bcf/Kconfig.defconfig

```diff
@@ -9,4 +9,14 @@ if BOARD_BEAGLECONNECT_FREEDOM
 config BOARD
 	default "beagleconnect_freedom"
 
+if SENSOR
+
+config OPT3001
+	default y
+
+config TI_HDC20XX
+	default y
```

Comment:
neither of these should be needed, they are enabled automatically once the corresponding devicetree entry is instantiated, this bit:

```
        default y
        depends on DT_HAS_TI_OPT3001_ENABLED
```

Can you drop them?